app/src/test/java/io/apicurio/registry/RegistryClientTest.java

```diff
@@ -44,7 +44,13 @@
 
     @RegistryServiceTest
     public void testSmoke(Supplier<RegistryService> supplier) {
-        supplier.get().deleteAllGlobalRules();
+        RegistryService service = supplier.get();
+
+        service.deleteAllGlobalRules();
+
+        Assertions.assertNotNull(service.toString());
+        Assertions.assertEquals(service.hashCode(), service.hashCode());
+        Assertions.assertEquals(service, service);
```

Comment:
These seem weird.  Can you add a comment explaining why this is happening?

Reply:
Just plain smoke test, so hash and equals work as expected.